src/api/java/de/teamlapen/vampirism/api/entity/factions/IFactionPlayerHandler.java

```diff
@@ -95,7 +95,7 @@
      * @param level
      * @return If successful
      */
-    boolean setFactionAndLevel(IPlayableFaction<? extends IFactionPlayer<?>> faction, int level);
+    boolean setFactionAndLevel(@Nonnull IPlayableFaction<? extends IFactionPlayer<?>> faction, int level);
```

Comment:
In the future we should try to include such unrelated changes in a commit/PR.
I am frequently doing the same atm, but it makes commits less clear, and merging more challenging.
Maybe Intellij's changelist feature can help to keep such unrelated changes separate and combine them to one larger cleanup commit.